core/dbt/flags.py

```diff
@@ -41,7 +41,7 @@
     "STATIC_PARSER": True,
     "WARN_ERROR": False,
     "WRITE_JSON": True,
-    "PARTIAL_PARSE": False,
+    "PARTIAL_PARSE": True,
```

Comment:
💯

Reply:
Yeah. Some parts of the code want the Z ending, and some parts don't, apparently. I moved the kludging of 'generated_at' into a post serialization method in BaseArtifactMetadata.